cmd/agent/app/run.go

```diff
@@ -277,6 +277,11 @@ func StartAgent() error {
 		log.Info("logs-agent disabled")
 	}
 
+	err = common.SetupSystemProbeConfig(sysProbeConfFilePath)
+	if err != nil {
+		log.Warnf("System probe config not read: %v", err)
```

Comment:
This should be an error log level, no ?

Reply:
I was unsure whether or not to include it as error or warn, since we do not include the system probe with some builds of the agent (puppy, currently windows). Happy to have it be error, but It would result in a situation where we would expect to get an error log on startup for certain agents. 

I'm unsure if that is generally okay.

Reply:
I see, I didn't thought of this. For me `SetupSystemProbeConfig` should not return any error with the puppy or windows agent.

Reply:
Mmm, okay, that makes sense - is there a standard way people have done this in the past? I haven't been able to find anything that is immediately apparent/intuitive for this.

Reply:
I'd actually suggest INFO here.  The system-probe.yaml doesn't *need* to be present if the system-probe isn't enabled.

Reply:
I kind of agree with @hush-hush notion that if the system probe is not enabled or supported and and we call `common.SetupSystemProbeConfig()` it should perhaps do nothing and just return `err == nil`? In which case the log level would make more sense as `Error`?

Reply:
@truthbk @hush-hush is there a way that we've programmatically checked for this to be able to return nil?

I've been looking around the agent and have not found anything besides perhaps a check to runtime which i'm not sure we want to do? If we did, I think it would make more sense to check in run.go and then not attempt to load it.